src/ontology/mondo-edit.obo

```diff
@@ -167008,7 +167010,7 @@ xref: Orphanet:177 {source="MONDO:subClassOf", source="OMIM:215100"}
 xref: Orphanet:309789 {source="OMIM:215100", source="MONDO:equivalentTo", source="DOID:0110851"}
 xref: UMLS:C1859133 {source="OMIM:215100", source="NCBI:mim2gene_medline", source="MONDO:equivalentTo", source="ORDO:309789/e", source="Orphanet:309789"}
 is_a: MONDO:0015776 {source="DOID:0110851", source="MONDO:Redundant", source="MONDOLEX:0008972", source="OMIM:215100", source="Orphanet:309789"} ! rhizomelic chondrodysplasia punctata
-is_a: MONDO:0019234 {source="MONDOLEX:0008972", source="OMIM:215100"} ! peroxisome biogenesis disorder
+is_a: MONDO:0100272 {source="https://github.com/monarch-initiative/mondo/issues/2632", source="https://www.clinicalgenome.org/affiliation/40049/"} ! peroxisome biogenesis disorder due to PEX7 defect
```

Comment:
seems odd to have an issue on github be the source of a subsumption.. Isnt this a rdfs:seeAlso? I don't know that but whatever it is, it should be standard.

Reply:
Happens multiple times throughout PR, just wondering if that is the accepted practice.